eeprom/eeprom.c

```diff
@@ -56,12 +56,12 @@ void write_log_entry(const char *str, uint8_t *index) {
         printf("Maximum log entries. Erasing the log to log the messages\n");
         erase_logs(index);
     }
-    size_t size_length = strlen(str); //not include NULL terminator
+    size_t size_length = strlen(str) + 1; //include NULL terminator
```

Comment:
This mysterious +/-1 could be named NULLC, or something of the sort, just as an answer for:
why is 1 added or subtraced here?

Reply:
Done